### PR TITLE
Fix masterclasses parsing failure

### PR DESCRIPTION
**Pull request by JonNorman**
## Background
We parse Eventbrite feeds to drive our Masterclasses and Live Events components. Currently, we expect each event to have at least one ticket that can be booked. It turns out that this was a _foolish_ assumption and it has indeed made an ass out of me.

## What does this change?
All properties derived from the `tickets` of an event are now optional.

Sneaky change - there is a logging statement that basically spams the logs with information we very very rarely care about. I've taken it out for now as it makes the logs a lot easier to grep/parse/read.

## What is the value of this and can you measure success?
Despite the fact that we will probably never serve one of these events to the end user, we will no longer get parsing failures when parsing these events.

## Does this affect other platforms - Amp, Apps, etc?
Nope

## Request for comment
@guardian/commercial-dev

### CODE REVIEW

File: commercial/app/model/merchandise/events/Eventbrite.scala [hunk 1]
```diff
@@ -130,22 +130,32 @@ object Eventbrite extends ExecutionContexts with Logging {
   trait TicketHandler {
     def tickets: Seq[Ticket]
 
-    val displayPriceRange = {
+    lazy val displayPriceRange: Option[String] = {
```
Comment (Calum-Campbell) [inline commercial/app/model/merchandise/events/Eventbrite.scala]: Why is this lazy?

Otherwise LGTM 👍

**Comment by regiskuckaertz** [comment]
Should the JSON endpoints be updated as well?

File: commercial/app/model/merchandise/events/Eventbrite.scala (hunk 1 above)
Comment (JonNorman) [inline commercial/app/model/merchandise/events/Eventbrite.scala]: Ah yes! Good spot. I originally did that in a brief spot of laziness before immediately abandoning the idea. My idea was to make it lazy and filter out all Events that have no tickets, but this was a terrible idea for many reason - forgot to delete the `lazy` though.

**Comment by JonNorman** [comment]
This only affects masterclasses (which we don't have an endpoint for) and live events. The live events JSON endpoint doesn't write these fields; that the logic is handled in the commercial-templates templates, as I believe it should be.

**tl;dr** this PR only affects the server-side rendering templates

**Comment by regiskuckaertz** [comment]
yahaaa 👍

**Comment by JonNorman** [comment]
[Masterclass feeds is failing.](https://trello.com/c/MLieooGy/200-masterclass-feeds-is-failing)
